docs/content/advanced-concepts.md

```diff
@@ -3,26 +3,16 @@ title: Advanced Concepts
 order: 2
 ---
 
-<a name="advanced-concepts"></a>
-
 # Advanced Concepts
 
-<a name="build--deployment"></a>
-
 ## Build & Deployment
 
 There are a variety of ways to host your Spectacle presentation.
 
-1. If you are integrating this lib yourself, take your build and follow the linked instructions from any of (but not limited to) this list of providers:
-
-   - [Heroku](https://devcenter.heroku.com/articles/git#deploying-code)
-   - [Zeit](https://zeit.co/docs/v2/platform/deployments)
-   - [Surge](https://surge.sh/help/deploying-continuously-using-git-hooks)
+1. If you are integrating this lib yourself, take your build and follow the linked instructions from any of (but not limited to) the following providers: [Heroku](https://devcenter.heroku.com/articles/git#deploying-code), [Zeit](https://zeit.co/docs/v2/platform/deployments), or [Surge](https://surge.sh/help/deploying-continuously-using-git-hooks).
```

Comment:
The formatting for this list of links was weird, so I opted to keep the links inline instead.